GiphySearch/GifCache.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MacOS 11.4
+//  Swift 5.0
+//  GiphySearch
+//
+
+import UIKit
+
+class GifCache {
+    
+    static var shared = GifCache()
+    
+    private init() { }
```

Comment:
Private `init` prevents API users to create their own instance of `GifCache`, we want to force use of shared instance